src/Runner.Common/BrokerServer.cs

```diff
@@ -92,7 +92,7 @@ public Task ForceRefreshConnection(VssCredentials credentials)
 
         public bool ShouldRetryException(Exception ex)
         {
-            if (ex is AccessDeniedException ade && ade.ErrorCode == 1)
+            if (ex is AccessDeniedException ade)
```

Comment:
Should we catch a specific exception type `RunnerVersionTooOldException`, since AccessDenied is ambiguous